docs/apm-package/data-streams.asciidoc

```diff
@@ -31,33 +31,31 @@ Traces::
 Traces are comprised of {apm-overview-ref-v}/apm-data-model.html[spans and transactions].
 Traces are stored in the following data stream:
 
-- Application traces: `traces-apm.<service.name>-<namespace>`
+- Application traces: `traces-apm-<namespace>`
 
 Metrics::
 
 Metrics include application-based metrics and basic system metrics.
 Metrics are stored in the following data streams:
 
-- Application defined metrics: `metrics-apm.<service.name>-<namespace>`
-- APM internal metrics: `metrics-apm.internal.<service.name>-<namespace>`
-- APM profiling metrics: `metrics-apm.profiling.<service.name>-<namespace>`
+- Application defined metrics: `metrics-apm.app.<service.name>-<namespace>`
+- APM internal metrics: `metrics-apm.internal-<namespace>`
+- APM profiling metrics: `metrics-apm.profiling-<namespace>`
 
 Logs::
 
 Logs include application error events and application logs.
 Logs are stored in the following data streams:
 
-- Application logs: `logs-<service.name>-<namespace>`
-- APM error/exception logging: `logs-apm.error.<service.name>-<namespace>`
+- Application logs: `logs-<namespace>`
```

Comment:
Should this instead be: `logs-apm-<namespace>`?

Reply:
we don't actually have this data stream yet, so this line can be deleted